.github/workflows/build-scan-push.yaml

```diff
@@ -129,3 +134,18 @@ jobs:
             ${{ steps.docker_meta.outputs.labels }}
             ```
           edit-mode: replace
+
+      # Compare the image built in the pull request with the one in production
+      - name: Docker Scout
+        id: docker-scout
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: docker/scout-action@v1
+        with:
+          command: compare
+          image: ${{ steps.docker_meta.outputs.tags }}
+          to: docker/${{ github.event.repository.name }}:latest
+          ignore-unchanged: true
+          only-severities: critical,high
+          organization: docker
+          write-comment: true
+          github-token: ${{ secrets.GITHUB_TOKEN }} # to be able to write the comment
```

Comment:
You may want to add `exit-code: true` to fail the build if vulnerability changes are detected.

Reply:
I don't remember I see that in the doc, probably need to update the doc :)